python/Ganga/Lib/Condor/Condor.py

```diff
@@ -542,8 +546,20 @@ def updateMonitoringInformation(jobs):
                         if exitCode.isdigit():
                             jobStatus = "completed"
                         else:
-                            logger.error("Problem extracting exit code from job %s. Line found was '%s'." % (
-                                jobDict[id].fqid, exitLine))
+                            # Some filesystems/setups have the file created but empty - only worry if it's been 10mins
+                            # since we first checked the file
+                            if len(lineList) == 0:
+                                if not jobDict[id].backend._stdout_check_time:
+                                    jobDict[id].backend._stdout_check_time = time.time()
+
+                                if (time.time() - jobDict[id].backend._stdout_check_time) < 10*60:
```

Comment:
OK, I hadn't understood it was that rare, no need to add yet another config option for this in that case :+1: